reference/recency-surveillance.v1.json

```diff
@@ -59,150 +59,155 @@
                       "encounter_uuid"
                     ],
                     "properties": {
+                      "description": "DHIS2 organisation unit uuid for this facility",
                       "dhis2_orgunit_uuid": {
                         "type": "string",
                         "default": "text"
                       },
-                      "referred_location": {
+                      "encounter_uuid": {
+                        "description": "Encounter universal unique identifier",
```

Comment:
@ODORA0 updated description, Encounter universal unique identifier usually a uuid for example c0d8ed6d-386a-4a1b-b9e7-af246fee0ca3

Reply:
@ssmusoke Done